lib/over_react.dart

```diff
@@ -48,7 +48,7 @@ export 'package:react/react.dart'
         SyntheticEventTypeHelpers,
         DataTransferHelper;
 
-export 'package:react/react_client/js_backed_map.dart' show JsBackedMap;
+export 'package:react/react_client/js_backed_map.dart' show JsBackedMap, JsMap;
```

Comment:
Hmm so we have to export JsMap since it's used for `@convertJsMapProp`... I'm not super in love with adding these to the main entrypoint.

Ooh wait, won't most consumers using those also be using `package:over_react/js_component.dart`, which already exports JsMap, unjsifyMapProp, jsifyMapProp, unjsifyRefProp, and jsifyRefProp?

If so, maybe we don't need to export them here?